### ISSUE TITLE
Warning...

### ISSUE THREAD

**Issue by tsoulii** [MaDHouSe79/mh-parking#41]
![image](https://github.com/MaDHouSe79/mh-parking/assets/34017026/cbab52f2-7446-44fb-a867-8b2ecbdfc820)

With every press of F5 this happens

![image](https://github.com/MaDHouSe79/mh-parking/assets/34017026/b7c6dcf7-d233-4c62-bd59-cc7e82c1cd46)

In addition to activating only two cars that do not work

**Comment by MaDHouSe79** [comment]
if you want limited parking you must use it as vip not as free for all..

**Comment by tsoulii** [comment]
Well, it's okay, I didn't notice about WARNING, which appears when riding in any car
It wasn't before, and I have a version of the server that does not contain mh-parking It doesn't have this problem

**Comment by MaDHouSe79** [comment]
as long this is a warning and not an error your fine, just ignore it 😉

**Comment by tsoulii** [comment]
Do you know something that I was one of the first people to try PARKING with you? :sweat_smile:  😀 
I hope you bring back Discord and I can help you get people into it
In general, thank you

**Comment by MaDHouSe79** [comment]
Lol i did not notice that i'm sorry😅 thanks for you help 👍 

***Bring back dicorde***
no i don't, bcause nobody was react on post or other stuff that i did, so i deleted the discord, cause i'm not the one who needs help with scripting 😉 

***Another example of why I did this***
Videos for scripts that i made, and post in discord, there is like 0 maby 3 likes on 600 sucscripbers and over 350 people in discord, so why sute i reallase any of my scripts that i have and i was happy to give to the comunnity, but the comunnity does nothing in return, a like or command or subscribe to my youtube channel so I can do more, not even a "thank you" if you solved someone's problem, so why should I still make scripts public and help people as people don't even appreciate what you give them for free, and as long as people don't show that they appreciate what I'm actually doing for then.

In short, I am not motivated enough to continue with this as I indicated above.

**Comment by tsoulii** [comment]
Yes yes I understand you 
It is not easy, even the most famous of them does not achieve what they want
I really appreciate what you did and I'm constantly watching to make sure you don't download anything new  :sweat_smile: :sweat_smile: 
I really appreciate what you did and I'm constantly watching to make sure you don't download anything new
Anyway, this is my discord tsouli_marouane
